assets-and-images.md

```diff
@@ -53,17 +57,30 @@ different locales or regions, reading directions, etc.
 
 ## Working with assets
 
-Your application can access its assets through the
-[AssetBundle](http://docs.flutter.io/flutter/services/AssetBundle-class.html)
-interface. The two main methods on an asset bundle allow you to load a string
+Your application can access its assets through as
+[AssetBundle](http://docs.flutter.io/flutter/services/AssetBundle-class.html).
+The two main methods on an asset bundle allow you to load a string
 (`loadString`) or an image (`loadImage`) out of the bundle, given a logical key.
 The logical key maps to the path to the asset specified in the `flutter.yaml`
-file at build time. The application's asset bundle is accessible through the
-[DefaultAssetBundle](http://docs.flutter.io/flutter/widgets/DefaultAssetBundle/of.html)
-widget's `of` method.
+file at build time.
+
+Each Flutter application has a
+[rootBundle](http://docs.flutter.io/flutter/services/rootBundle.html)
+for easy access to the main asset bundle. You can use the `rootBundle`
+from `package:flutter/services.dart`
+to easily access configuration file assets or assets that don't need to be
+aware of a `BuildContext`.
+
+For example, use the `rootBundle` to easily
+load a JSON file asset.
+
+    String assetContents = await rootBundle.loadString('assets/config.json');
+
+You can also access a context-aware `AssetBundle` with
```

Comment:
That's fair, but I don't understand why I want an asset bundle associated with a context. Nor do I know how to associate a bundle with a context. So it's hard for me to sell the longer `DefaultAssetBundle.of(context).loadString(...)`